lib/namespace/resource.go

```diff
@@ -0,0 +1,12 @@
+package namespace
```

Comment:
nit: a bunch of files in this package are missing the copyright notice

Reply:
there is only one file in this package.  You mean another package?

Reply:
oo, I mean teh options package